frontend/src/components/ManagePermissionsTable/index.vue

```diff
@@ -0,0 +1,632 @@
+    AdminRoleAuthGroup,
+    FamAccessControlPrivilegeGetResponse,
+    FamAppAdminGetResponse,
+} from "fam-admin-mgmt-api/model";
+import type { FamApplicationUserRoleAssignmentGetSchema } from "fam-app-acsctl-api/model";
+
+import TableToolbar from "@/components/Table/TableToolbar.vue";
+import TableHeaderTitle from "@/components/Table/TableHeaderTitle.vue";
+import TableSkeleton from "@/components/Skeletons/TableSkeleton.vue";
+import ErrorText from "@/components/UI/ErrorText.vue";
+import Chip from "@/components/UI/Chip.vue";
+import {
+    AdminMgmtApiService,
+    AppActlApiService,
+} from "@/services/ApiServiceFactory";
+import { EnvironmentSettings } from "@/services/EnvironmentSettings";
+import { formatUserNameAndId } from "@/utils/UserUtils";
+import {
+    DEFAULT_ROW_PER_PAGE,
+    TABLE_ROWS_PER_PAGE,
+    TABLE_PAGINATOR_TEMPLATE,
+    TABLE_CURRENT_PAGE_REPORT_TEMPLATE,
+} from "@/store/Constants";
+import { formatAxiosError } from "@/utils/ApiUtils";
+import {
+    AddAppPermissionRoute,
+    AddFamPermissionRoute,
+    UserDetailsRoute,
+} from "@/router/routes";
+import { selectedApp } from "@/store/ApplicationState";
+import type { PermissionNotificationType } from "@/types/ManagePermissionsTypes";
+import { NewFamAdminQueryParamKey } from "@/views/AddFamPermission/utils";
+import {
+    NewAppAdminQueryParamKey,
+    NewDelegatedAddminQueryParamKey,
+} from "@/views/AddAppPermission/utils";
+
+import NewUserTag from "./NewUserTag.vue";
+import ConfirmDialogText from "./ConfirmDialogText.vue";
+import {
+    getTableHeaderDescription,
+    getTableHeaderTitle,
+    getGrantButtonLabel,
+    filterList,
+    getHeaders,
+    type ConfirmTextType,
+    createNotification,
+    deleteAppAdminContext,
+    deleteDelegatedAdminContext,
+    deleteFamPermissionContext,
+    NEW_ACCESS_STYLE_IN_TABLE,
+} from "./utils";
+
+const router = useRouter();
+const route = useRoute();
+
+const environment = new EnvironmentSettings();
+
+const newFamAdminIds = route.query[NewFamAdminQueryParamKey]
+    ? String(route.query[NewFamAdminQueryParamKey]).split(",").map(Number)
+    : [];
+
+const newAppUserIds = route.query[NewAppAdminQueryParamKey]
+    ? String(route.query[NewAppAdminQueryParamKey]).split(",").map(Number)
+    : [];
+
+const newDelegatedAdminIds = route.query[NewDelegatedAddminQueryParamKey]
+    ? String(route.query[NewDelegatedAddminQueryParamKey])
+          .split(",")
+          .map(Number)
+    : [];
+
+const props = defineProps<{
+    authGroup: AdminRoleAuthGroup;
+    appName: string;
+    appId: number;
+    addNotifications: (newNotifications: PermissionNotificationType[]) => void;
+}>();
+
+// Fam App Admins data query
+const appAdminQuery = useQuery({
+    queryKey: ["application_admins"],
+    queryFn: () =>
+        AdminMgmtApiService.applicationAdminApi
+            .getApplicationAdmins()
+            .then((res) => res.data),
+    refetchOnMount: "always",
+    enabled: props.authGroup === "FAM_ADMIN",
```

Comment:
https://github.com/bcgov/nr-forests-access-management/pull/1650/commits/7156e7b3f59aa1796afb1247867c9b38eabcb648

Reply:
Does this work for you with `enum` so that you only need to hardcode the values once in enum and you also get to have your favorite typing 'ManagePermissionsTableType' :) ?
```
export enum MANAGE_PERMSN_TBL_ENUM {
    FAM_APP_ADMIN = "FAM_APP_ADMIN",
    DELEGATED_ADMIN = "DELEGATED_ADMIN",
    APP_USER = "APP_USER",
}

export type ManagePermissionsTableType =
    | MANAGE_PERMSN_TBL_ENUM.FAM_APP_ADMIN
    | MANAGE_PERMSN_TBL_ENUM.DELEGATED_ADMIN
    | MANAGE_PERMSN_TBL_ENUM.APP_USER;
```
![image](https://github.com/user-attachments/assets/c3da97f4-6056-4a29-b0d8-6f449b66be16)

Reply:
https://github.com/bcgov/nr-forests-access-management/pull/1650/commits/1647b0ba31143fab43f93b7f79a44741925554ee

I was not a big fan of Enum, but i see its benefit here now